Configs/.config/hypr/hypridle.conf

```diff
@@ -0,0 +1,39 @@
+$SCREENLOCK = hyprlock # If you prefer to use swaylock: 'swaylock -f'
```

Comment:
I assume this has to go along with hyprlock PR?

Reply:
Yes sure. Users can reference this PR  for now.